frontend/src/app/bounty/create/page.tsx

```diff
@@ -57,7 +54,17 @@ const CreateBountyPage: FC = () => {
     const [imgLink, setImgLink] = useState("");
     const [filename, setFilename] = useState<string | undefined>();
 
+    const [minDeadline, setMinDeadline] = useState<Date>(new Date());
+    const blockTimeStamp = useBlockTimestamp();
```

Comment:
In `camelCase` we only capitalize the first letter of words (except the first one). Since we often write "timestamp" as one word, there is no need to capitalize the letter "s".

```suggestion
    const blockTimestamp = useBlockTimestamp();
```